definitions/ext-host/definition.yml

```diff
@@ -10,9 +10,20 @@ synthesis:
         - attribute: metricName
           prefix: datadog.system.
 
+    # telemetry from generic flex agent
+    - identifier: displayName
+      name: displayName
+      encodeIdentifierInGUID: true
+      conditions:
+      - attribute: eventType
+        value: FlexSystemSample
+
 dashboardTemplates:
-  newRelic:
-    template: dashboard.json
+  flexagent:
+    template: flex-dashboard.json
+  datadog:
```

Comment:
@naxhh will we have issues here given the datadog telemetry may not have the instrumentation.provider attribute?

Reply:
I was checking the data and you are right indeed. Datadog agent doesn't provide a inst.provider.

Put it first in all rules so it will default to that one when the instrumentation.provider doesn't exist

Reply:
@naxhh have moved it first, would it be okay to merge now?